R/entity-attribute-value.R

```diff
@@ -659,3 +662,70 @@ eav_encode_covariate <- function(pdata, covariate_def, aname = "variable") {
   #args <- covariate_def[['arguments']] # Let us assume is a list of character(1)s for now
   encode.fn(pdata[[aname]])
 }
+
+##########
+
+cast_covariate.new <- function(covariate, values, cov.def, .fds) {
```

Comment:
cast_covariate_new would be better. We use S3 in this package, so it would be better to avoid confusion.